.history/force-app/main/default/classes/Constants_20210713163336.cls

```diff
@@ -0,0 +1,10 @@
+/**
+ * @description       : 
+ * @author            : Mikołaj Lechtański
+ * @last modified on  : 13.07.2021
+**/
+public with sharing class Constants {
```

Comment:
2 klasy z roznymi nazwami plikow ale z ta sama nazwa klasy to chyba sie nie kompiluje

Reply:
dodałem niepotrzebniej jakąś wtyczkę gitową i powrzucało historyczne pliki z folderu ".history". Naprawię to przez .gitignore